### ISSUE TITLE
Transpose Matrix cells

### ISSUE THREAD

**Issue by NMO13** [Caleydo/malevo#101]
When using our heatmap approach to show multiple conditions, it is easy to find visual patterns in vertical direction because the epochs are aligned in horizontal direction. However finding visual patterns in horizontal direction is not so easy.
E.g. the most right blue bars are hard to compare over multiple cells:
![image](https://user-images.githubusercontent.com/3988444/38026809-ce095e52-328d-11e8-9a79-6d8e2f607055.png)
Rotating both cells by 90 degree would make it easier:
![image](https://user-images.githubusercontent.com/3988444/38027306-7d26d9f4-328f-11e8-8059-71b9b757d41a.png)

**Comment by mstreit** [comment]
I totally agree. To me it seems that this would be something the user should be able to toggle on demand. Before we implement this, we need to be sure that we keep the heatmap encoding.

Btw: If we allow the user to switch, we need a legend that explain in which direction the epochs progress.

**Comment by thinkh** [comment]
Ad legend: Yes, I agree with that. A possible place for the legend would be the currently unused upper left corner of the conf matrix. The legend could also act as toggle button.

**Comment by mstreit** [comment]
yes, that's a good idea. possible icon for that: https://fontawesome.com/icons/redo-alt?style=solid
